subprojects/docs/src/docs/userguide/core-plugins/build_init_plugin.adoc

```diff
@@ -80,6 +80,8 @@ You can change the package used for generated source files using the `--package`
 
 If the `--incubating` option is provided, Gradle will generate build scripts which may use the latest versions of APIs, which are marked `@Incubating` and remain <<feature_lifecycle.adoc#feature_lifecycle,subject to change>>.
 
+If the `--overwrite` option is provided, Gradle will have permission to overwrite any existing files in the directory where the `init` task is run. Otherwise, `init` will fail if any files are found in the directory.
```

Comment:
```suggestion
If the `--overwrite` option is provided, Gradle overwrites existing files in the directory where the `init` task runs. By default, `init` fails if Gradle finds any files in the directory.
```
Suggestion: trim some words, make the default behavior more obviously "the default", revise passive voice to active voice where it doesn't clash with adjacent sentence structures.

Reply:
Thanks @nathan-contino, updated documentation with these suggestions.